modules/ROOT/pages/apply-configure-xml-threat-task.adoc

```diff
@@ -9,31 +9,34 @@ endif::[]
 >s|Policy Name|XML Threat Protection
 >s|Summary|Protects against malicious XML in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
```

Comment:
"First Mule version available"?

Reply:
We have been doing "Available from Mule version" in other policies. Should we stick to that?